### ISSUE TITLE
Add option to exclude tracks based on pattern matching

### ISSUE THREAD

**Issue by NeilN1** [web-scrobbler/web-scrobbler#2222]
Add two fields (one for artist, one for track title) a user would use to enter lists of artists and track titles. A track having info matching the artist or track title would not be scrobbled.

This would stop the scrobbling of promos/DJ chatter some radio-like services play.  Example from SiriusXM: Artist: @SiriusXMHits1 Track: @MorningMashUp

**Comment by alexesprit** [comment]
@NeilN1 We already have a simple rule for SiriusXM. Instead of adding new options, I suggest to add excludes directly to SiriusXM connector. The advantage of this way is it will work for ALL users w/o any configuration. I'm pretty sure no one wants to scrobble non-music stuff.

> This would stop the scrobbling of promos/DJ chatter some radio-like services play. Example from SiriusXM: Artist: @SiriusXMHits1 Track: @MorningMashUp

This could be fixed by adding a rule like this:
> If an artist name and (maybe or?) a track title start with "@" symbol, then ignore this track.

If there're additional things should be skipped, please let me know.

**Comment by NeilN1** [comment]
Fair enough. I didn't want to bother the developers with constant exclusion requests so I suggested something user based.  I will keep an eye out for other examples and post here (let me know if there's somewhere else you'd prefer I send the requests).

**Comment by seafea** [comment]
ignoring track titles that start with @ would filter out a good chunk of them, but there's still some like "w/ @kadeshow" or "Wednesday DL"

Could we maybe have a list of known DJ / promo tracks to specifically exclude? I could maybe help out with this one.

**Comment by alexesprit** [comment]
@seafea yes, feel free to post this list here. I will add it to the SiriusXM connector.

**Comment by JunkiEDM** [comment]
+1, I frequently use Soundcloud for listening but had to stop using this extension because it kept scrobbling tracks & artists that I don't want scrobbled.
